package.json

```diff
@@ -2,7 +2,8 @@
     "devDependencies": {
         "@rocket.chat/apps-engine": "git+https://github.com/WideChat/Rocket.Chat.Apps-engine.git#master",
         "@types/node": "^10.17.24",
-        "tslint": "^5.10.0",
+        "prettier": "^2.0.5",
```

Comment:
I think this is not required and everyone might not be using this.

Reply:
Yes, everyone might not be using it. I saw this dependency update from upstream in the Dialogflow repo. That's why I have updated it.